cs/cs.sln

```diff
@@ -258,6 +258,23 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "output-no-slash", "test\cod
 		{92915BD9-8AB1-4E4D-A2AC-95BBF4F82D89} = {92915BD9-8AB1-4E4D-A2AC-95BBF4F82D89}
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "grpc_pingpong", "..\examples\cs\comm\grpc_pingpong\grpc_pingpong.csproj", "{618420F1-6465-4DF4-BD1B-747765F481FF}"
+	ProjectSection(ProjectDependencies) = postProject
+		{43CBBA9B-C4BC-4E64-8733-7B72562D2E91} = {43CBBA9B-C4BC-4E64-8733-7B72562D2E91}
+		{21E175D5-BBDD-4B63-8FB7-38899BF2F9D1} = {21E175D5-BBDD-4B63-8FB7-38899BF2F9D1}
+		{AF03BAE6-2470-4E1B-A683-3EBDCDC595FA} = {AF03BAE6-2470-4E1B-A683-3EBDCDC595FA}
+	EndProjectSection
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "grpc", "src\grpc\grpc.csproj", "{AF03BAE6-2470-4E1B-A683-3EBDCDC595FA}"
+	ProjectSection(ProjectDependencies) = postProject
+		{40452223-3A34-4841-9A69-E0B6640B4108} = {40452223-3A34-4841-9A69-E0B6640B4108}
+		{2E6E238C-9017-445C-9611-66DA780609BE} = {2E6E238C-9017-445C-9611-66DA780609BE}
+		{43CBBA9B-C4BC-4E64-8733-7B72562D2E91} = {43CBBA9B-C4BC-4E64-8733-7B72562D2E91}
+		{92915BD9-8AB1-4E4D-A2AC-95BBF4F82D89} = {92915BD9-8AB1-4E4D-A2AC-95BBF4F82D89}
+	EndProjectSection
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "grpc", "test\grpc\grpc.csproj", "{D5F087E7-1EE9-4AEB-83AF-5F234C69B867}"
```

Comment:
test\grpc\grpc.csproj is missing solution dependencies on Bond.Attributes, Bond.Core, Bond.Reflection, and Bond.IO.